pkg/kubectl/resource_printer.go

```diff
@@ -337,20 +337,43 @@ type PrintOptions struct {
 // received from watches.
 type HumanReadablePrinter struct {
 	handlerMap map[reflect.Type]*handlerEntry
-	Options    PrintOptions
+	options    PrintOptions
 	lastType   reflect.Type
 }
 
 // NewHumanReadablePrinter creates a HumanReadablePrinter.
-func NewHumanReadablePrinter(options PrintOptions) *HumanReadablePrinter {
+func NewHumanReadablePrinter(pOptions PrintOptions) *HumanReadablePrinter {
 	printer := &HumanReadablePrinter{
 		handlerMap: make(map[reflect.Type]*handlerEntry),
-		Options:    options,
+		options:    pOptions,
 	}
 	printer.addDefaultHandlers()
 	return printer
 }
 
+// formatResourceName receives a resource kind, name, and boolean specifying
+// whether or not to update the current name to "kind/name"
+func formatResourceName(kind, name string, withKind bool) string {
+	if !withKind || kind == "" {
+		return name
+	}
+
+	return kind + "/" + name
+}
+
+// GetResourceName returns the currently set resource name
+func (h *HumanReadablePrinter) GetResourceName() string {
```

Comment:
Talking about names, are there other parts of the code base talking about
"kind"? It sounded okay when I saw it in your previous PR, but since then
I've been noticing the CLI talks about "type", resource type.

On Aug 4, 2016 5:44 PM, "Juan Vallejo" <notifications@github.com> wrote:

> In pkg/kubectl/resource_printer.go
> <https://github.com/kubernetes/kubernetes/pull/28509#discussion_r73547923>
> :
>
> >  	}
> >  	printer.addDefaultHandlers()
> >  	return printer
> >  }
> >
> > +// formatResourceName receives a resource kind, name, and boolean specifying
> > +// whether or not to update the current name to "kind/name"
> > +func formatResourceName(kind, name string, withKind bool) string {
> > +	if !withKind || kind == "" {
> > +		return name
> > +	}
> > +
> > +	return kind + "/" + name
> > +}
> > +
> > +// GetResourceName returns the currently set resource name
> > +func (h *HumanReadablePrinter) GetResourceName() string {
>
> Ah thanks for catching this. I will rename this method to GetResourceKind,
> as name and kind are not the same thing
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28509/files/9c6a42c940f9b40e6347cc5397e1f1cb9890964f#r73547923>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFa8xO19LKOvcK_PjMMVdk9gh5jQaCwks5qcgj3gaJpZM4JFgN2>
> .
>